setup.py

```diff
@@ -17,6 +17,9 @@
     'tabulate>=0.8.2',
     'transitions>=0.7.1',
     'psycopg2-binary==2.8.6',
+    # Compliance-checker 5.0.1 have problem with numpy.float / numpy.int
+    # Starting 1.24.0 numpy.float / numpy.int no longer support and should use float / int instead.
+    'numpy<1.24.0',
```

Comment:
Ok, but we have a separate [constraints.txt](https://github.com/aodn/python-aodncore/blob/master/constraints.txt) file for such conditions, so this should really go in there. Those constraints are automatically included by commands like `pip install -r requirements.txt`